assets/scss/modules/_buttons.scss

```diff
@@ -3,30 +3,36 @@
     display: inline-block;
     text-align: center;
     text-decoration: none;
-    text-transform: uppercase;
     border: none;
     outline: none;
-    margin: 0 5px;
     appearance: none;
+    min-width: 140px;
     font-family: $dp-font-family-proximanova;
     border-radius: $dp-border-radius;
-    @include transition;
+    @include transition-all;
 
-    &.button-big {
+    .ms-icon {
+      font-size: 20px;
```

Comment:
Use variables plz

Reply:
You have this in a variable `$dp-form-font-size` in https://github.com/FromDoppler/Doppler-UI-System/blob/master/assets/scss/core/_typography.scss#L25 please check if is semantically correct to use it here.